### DISCUSSION TITLE
Fluent generator. Column ordering

### DISCUSSION THREAD

**Discussion by usdaud** [linq2db/linq2db#4024] [Ideas]
Seems with that style

```
builder
				.Entity<TestEntity>()
				.IsColumnRequired()
				.Property(m => m.Column1)
				.Property(m => m.Column2);

			table = db.CreateTable<TestEntity>();
```

column order isn't determined and need to use HasOrder method.

For fluent description will be more friendly to use the same ordering as method Property calling.

**Comment by MaceWindu** [comment]
Please provide full sample. Your code miss Build call and it is not clear wether you pass configured mapping schema to context

**Comment by MaceWindu** [comment]
no response received
